soc/nordic/nrf53/Kconfig

```diff
@@ -208,7 +208,7 @@ config SOC_NRF53_CPUNET_MGMT
 
 config SOC_NRF53_CPUNET_ENABLE
 	bool "NRF53 Network MCU is enabled at boot time"
-	default y if NRF_802154_SER_HOST
+	default y if NRF_802154_SER_HOST || BT
```

Comment:
rather than no-up here, put these in nrf/soc/nordic/nrf5* folders (can just make a Kconfig.defconfig and put e.g.
```
config SOC_NRF53_CPUNET_ENABLE
	default y if BT
```
)